### PR TITLE
Dialogue fix for Karolis Springs, Letas Stream, Sienakal Graveyard and Carlyle's Mausoleum, and other fixes

### PR DESCRIPTION
**Pull request by InfinityZeroX**
Dialogue fix for Karolis Springs
Dialogue fix for Letas Stream
Dialogue fix for Sienakal Graveyard
Dialogue fix for Carlyle's Mausoleum
Some grammer fixes here and there
Removal of double spacing between words in QUEST_LV_0100.tsv
Removal of spacing at the end of item names and their descriptions
Changed all "advices" to "advice"
Changed some "sacred object" to "holy relic"

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge. :+1:

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -11240,7 +11240,7 @@ QUEST_LV_0100_20151224_011239	I might have a request for you.{nl}You might be th
 QUEST_LV_0100_20151224_011240	This job would be perfect for you.{nl}I do have a keen eye for people.
 QUEST_LV_0100_20151224_011241	The client should be pleased by your performance.{nl}Well done.
 QUEST_LV_0100_20151224_011242	You didn't see him?{nl}His mind is not all that sound. Where could he be?{nl}
-QUEST_LV_0100_20151224_011243	Where's my manners, allow me to introduce myself.{nl}I am studying the Royal Mausoleum of the Great King Zachariel along with the historians.
+QUEST_LV_0100_20151224_011243	Where're my manners, allow me to introduce myself.{nl}I am studying the Royal Mausoleum of the Great King Zachariel along with the historians.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:11243]: "Where're"

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -8739,8 +8739,8 @@ QUEST_LV_0100_20151001_008738	I met the grave keeper while working on purifying
 QUEST_LV_0100_20151001_008739	The grave keeper stands at the end of Vepavi Corridor. {nl}Can you ask him about the sacred object of the goddess for me? 
 QUEST_LV_0100_20151001_008740	A sacred object of the goddess? {nl}What? Are you teaming up with that rude Bokor? {nl}
 QUEST_LV_0100_20151001_008741	Get out of this grave site. {nl}Just thinking about that Bokor brings back my anger. {nl}
-QUEST_LV_0100_20151001_008742	That Bokor went around breaking the coffins of the dead saying they rise back! {nl}I don't understand how he can do such a thing with a sane mind!
-QUEST_LV_0100_20151001_008743	sacred object that you can't even approach... {nl}I understand if they block the monsters or demons from getting close to it but even humans, that I don't understand.
+QUEST_LV_0100_20151001_008742	That Bokor went around breaking the coffins of the dead saying they rise back! {nl}I don't understand how she can do such a thing with a sane mind!
+QUEST_LV_0100_20151001_008743	A sacred object that you can't even approach... {nl}I understand if they block the monsters or demons from getting close to it but even humans, that I don't understand.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:8743]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -8751,20 +8751,20 @@ QUEST_LV_0100_20151001_008750	Burying it back in the ground is not all. {nl}Pay
 QUEST_LV_0100_20151001_008751	What are you waiting for! Move quickly! {nl}I cannot... handle it anymore. 
 QUEST_LV_0100_20151001_008752	Sorry for being sharp a while ago. I was a bit sensitive. {nl}Actually, having to point the blades to people whom I should {nl}take care of as a grave keeper is not something I am comfortable with. {nl}
 QUEST_LV_0100_20151001_008753	Thank you for giving them peace for me. 
-QUEST_LV_0100_20151001_008754	Anyway, I know of the sacred object of the goddess that the Bokor mentioned.{nl}Those people from Order of the Tree of Truth made it one day. {nl}
+QUEST_LV_0100_20151001_008754	Anyway, I know of the sacred object of the goddess that the Bokor mentioned.{nl}Those people from the Order of the Tree of Truth made it one day. {nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:8754]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -9440,10 +9440,10 @@ QUEST_LV_0100_20151001_009439	Hand over the Grave Keeper's Spell Doll to Sigis
...
 QUEST_LV_0100_20151001_009442	Investigate the goddess' sacred object which was created by the Order of the Tree of Truth
-QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielva Pray Room.
+QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielvar Prayer Room.
 QUEST_LV_0100_20151001_009444	You've destroyed the barrier, but you got near the sacred object, the spell doll was destroyed. Return to Sigis and tell him about it.
 QUEST_LV_0100_20151001_009445	Sigis asked you to tell Bokor Juta that the spell doll was destroyed.
-QUEST_LV_0100_20151001_009446	Investigate the sacred object which was created by Order of the Tree of Truth
+QUEST_LV_0100_20151001_009446	Investigate the sacred object which was created by the Order of the Tree of Truth
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:9446]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -9440,10 +9440,10 @@ QUEST_LV_0100_20151001_009439	Hand over the Grave Keeper's Spell Doll to Sigis
 QUEST_LV_0100_20151001_009440	You've found the Grave Keeper's Spell Doll. Hand it over to Sigis.
 QUEST_LV_0100_20151001_009441	You've engraved the trace of the evil magic onto the Grave Keeper's Spell Doll. Ask Sigis how to use the spell doll.
 QUEST_LV_0100_20151001_009442	Investigate the goddess' sacred object which was created by the Order of the Tree of Truth
-QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielva Pray Room.
+QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielvar Prayer Room.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:9443]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -9440,10 +9440,10 @@ QUEST_LV_0100_20151001_009439	Hand over the Grave Keeper's Spell Doll to Sigis
 QUEST_LV_0100_20151001_009440	You've found the Grave Keeper's Spell Doll. Hand it over to Sigis.
 QUEST_LV_0100_20151001_009441	You've engraved the trace of the evil magic onto the Grave Keeper's Spell Doll. Ask Sigis how to use the spell doll.
 QUEST_LV_0100_20151001_009442	Investigate the goddess' sacred object which was created by the Order of the Tree of Truth
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:9442]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -9440,10 +9440,10 @@ QUEST_LV_0100_20151001_009439	Hand over the Grave Keeper's Spell Doll to Sigis
 QUEST_LV_0100_20151001_009440	You've found the Grave Keeper's Spell Doll. Hand it over to Sigis.
 QUEST_LV_0100_20151001_009441	You've engraved the trace of the evil magic onto the Grave Keeper's Spell Doll. Ask Sigis how to use the spell doll.
 QUEST_LV_0100_20151001_009442	Investigate the goddess' sacred object which was created by the Order of the Tree of Truth
-QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielva Pray Room.
+QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielvar Prayer Room.
 QUEST_LV_0100_20151001_009444	You've destroyed the barrier, but you got near the sacred object, the spell doll was destroyed. Return to Sigis and tell him about it.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:9444]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -9440,10 +9440,10 @@ QUEST_LV_0100_20151001_009439	Hand over the Grave Keeper's Spell Doll to Sigis
...
+QUEST_LV_0100_20151001_009443	You are ready to look at the sacred object of the goddess which was created by the Believers of the Order of the Tree of Truth. Bring the spell doll and check the sacred object of the goddess at Sielvar Prayer Room.
 QUEST_LV_0100_20151001_009444	You've destroyed the barrier, but you got near the sacred object, the spell doll was destroyed. Return to Sigis and tell him about it.
 QUEST_LV_0100_20151001_009445	Sigis asked you to tell Bokor Juta that the spell doll was destroyed.
-QUEST_LV_0100_20151001_009446	Investigate the sacred object which was created by Order of the Tree of Truth
+QUEST_LV_0100_20151001_009446	Investigate the sacred object which was created by the Order of the Tree of Truth
 QUEST_LV_0100_20151001_009447	Juta told you that he will take a look at it himself as he heard the barrier was unlocked. Go there first and wait for Juta.
 QUEST_LV_0100_20151001_009448	You've defeated Abomination that appeared from the sacred object of the goddess. Talk to Juta.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:9448]: sacred object -> holy relic

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -9481,18 +9481,18 @@ QUEST_LV_0100_20151001_009480	Defeat Margiris that has shown its true nature
 QUEST_LV_0100_20151001_009481	Margiris has shown his true nature as he saw the sacred object of the Order being destroyed. Defeat him.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:9481]: sacred object -> holy relic

File: EnglishTranslation-master/SKILL.tsv [hunk 10]
```diff
@@ -4959,7 +4959,7 @@ SKILL_20150729_004958	Maximum Amount: +#{SkillFactor}#{nl}Duration: 200 seconds
 SKILL_20150729_004959	Maximum Duration: #{CaptionRatio2}# seconds{nl}Consumes #{CaptionRatio}#% SP per second
 SKILL_20150729_004960	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Maximum Targets: #{CaptionRatio2}#
 SKILL_20150729_004961	STR, CON, INT, SPR, DEX: -#{CaptionRatio}#{nl}Consumes 3% SP per 3 seconds{nl}Maximum Duration: 10 seconds
-SKILL_20150729_004962	Transmits #{CaptionRatio}# of INT
+SKILL_20150729_004962	Transmits #{CaptionRatio}#% of INT
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: `Transmits #{CaptionRatio}#% INT`

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -7776,18 +7776,18 @@ QUEST_LV_0100_20150803_007775	The voice had a powerful dark energy.{nl}You bette
 QUEST_LV_0100_20150803_007776	Old Manager
 QUEST_LV_0100_20150803_007777	Resentful Spirit
 QUEST_LV_0100_20150803_007778	Don't worry. I'll give you the recipe for the antidote.{nl}Ridimed leaves from there will be enough to make it.
-QUEST_LV_0100_20150803_007779	The grave keeper who manages here, Catacombs may know a little about that sacred object and the barrier.{nl}But, I can't talk to the grave keeper.{nl}
-QUEST_LV_0100_20150803_007780	The grave keeper shouted at me while I was preparing for the ritual here.{nl}Can you ask about the sacred object of the goddess to the grave keeper on behalf of me?
+QUEST_LV_0100_20150803_007779	The grave keeper who manages here, Catacombs may know a little about that holy relic and the barrier.{nl}But, I can't talk to the grave keeper.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:7779]: The grave who manages the catacombs here may know a little ....

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**